cves/kernel/CVE-2021-29648.yml

```diff
@@ -326,8 +339,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: Three developers signed off on the fix commit
```

Comment:
It may be helpful to name the developers

Reply:
Thanks. We were told to not directly list names so I put down their roles instead